Directory Tree Generator/tree.py

```diff
@@ -0,0 +1,46 @@
+#Directory Tree Generator
+
+import os
+
+def realname(path, root=None):
+    if root is not None:
+        path=os.path.join(root, path)
+    result=os.path.basename(path)
+    if os.path.islink(path):
+        realpath=os.readlink(path)
+        result= '%s -> %s' % (os.path.basename(path), realpath)
+    return result
+
+def ptree(startpath, depth=-1):
+    prefix=0
```

Comment:
```suggestion
    prefix=0
    assert os.path.isdir(startpath),"Directory not valid"
```

Reply:
changes are done and you can review it now